pkg/clusterconditions/promql/promql.go

```diff
@@ -20,32 +21,38 @@ import (
 	"k8s.io/klog/v2"
 
 	"github.com/openshift/cluster-version-operator/pkg/clusterconditions/cache"
+	"github.com/openshift/cluster-version-operator/pkg/cvo/options"
 )
 
 // PromQL implements a cluster condition that matches based on PromQL.
 type PromQL struct {
 	kubeClient kubernetes.Interface
 
+	QueryNamespace string
+	QueryService   string
```

Comment:
do these need to be public?

Reply:
No, they don't have to be, thanks for noticing.

Reply:
Fixed in the commit. Thanks for noticing, everybody.